optimum/exporters/tasks.py

```diff
@@ -1626,7 +1731,7 @@ def _infer_task_from_model_name_or_path(
                 # zero-shot-object-detection or object-detection.
                 if pipeline_tag is not None and pipeline_tag not in ["conversational", "object-detection"]:
                     inferred_task_name = TasksManager.map_from_synonym(model_info.pipeline_tag)
-                else:
+                elif library_name == "transformers":
```

Comment:
So diffusers is handled by `inferred_task_name = TasksManager.map_from_synonym(model_info.pipeline_tag)`?

Reply:
Yes when pipeline tag is not None. will add another case when `diffusers` is the lib but pipeline tag is not defined.